src/open_inwoner/haalcentraal/signals.py

```diff
@@ -17,58 +18,83 @@
 logger = logging.getLogger(__name__)
 
 
-def fetch_data(instance):
+def fetch_data(instance, brp_version):
     config = HaalCentraalConfig.get_solo()
 
     if not config.service:
         logger.warning("no service defined for Haal Centraal")
         return {}
 
     client = config.service.build_client()
-    url = urljoin(client.base_url, "personen")
-
-    try:
-        data = client.operation(
-            operation_id="GetPersonen",
-            url=url,
-            data={
-                "fields": "naam,geboorte",
-                "type": "RaadpleegMetBurgerservicenummer",
-                "burgerservicenummer": [instance.bsn],
-            },
-            request_kwargs=dict(
-                headers={"Accept": "application/hal+json"},
-            ),
-        )
-    except RequestException as e:
-        logger.exception("exception while making request", exc_info=e)
-        return {}
-    except ClientError as e:
-        logger.exception("exception while making request", exc_info=e)
-        return {}
+
+    if brp_version == "ENSCHEDE" or brp_version == "DEFAULT":
```

Comment:
I'd make this:
 if == "2.0
 ...
 elif == "1.3"

So avoid naming installations, we're targetting two versions of Haal Centraal BRP so lets use the version numbers to differentiate between them